charts/fluentd-elasticsearch/README.md

```diff
@@ -48,6 +48,7 @@ The following table lists the configurable parameters of the Fluentd elasticsear
 | `annotations`                        | Optional daemonset annotations                                                 | `NULL`                                 |
 | `podAnnotations`                     | Optional daemonset's pods annotations                                          | `NULL`                                 |
 | `configMaps`                         | Fluentd configmaps                                                             | `default conf files`                   |
+| `aws`                                | Enable AWS request signing sidecar                                             | `false`                                |
```

Comment:
Could we rename this var to something more meaningful please?
Maybe something like awsSigningSidecar?

Reply:
Renamed.